README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.0.0
```

Comment:
The code changes involve updating the version number from 4.0.0 to 4.1.0 for testing semantic versioning. This appears to be a minor version update, which suggests that there may be small improvements or bug fixes in the new version. Overall, the code changes seem straightforward and do not raise any immediate concerns.